### ISSUE TITLE
it will be great to have ability to see "named blocks" usage

### ISSUE THREAD

**Issue by lifeart** [emberjs/ember-inspector#1472]
it will be great to have ability to see "named blocks" usage in component explorer

why? 

in ember v3.25.0-beta.1 named blocks enabled

https://github.com/emberjs/ember.js/pull/19318

https://github.com/emberjs/rfcs/blob/master/text/0460-yieldable-named-blocks.md

**Comment by rwwagner90** [comment]
@lifeart how exactly would you like this feature to work?

**Comment by lifeart** [comment]
@rwwagner90 I think it may be `<:slotName><InternalComponentIfExists /></:slotName>` in component explorer, if there is no components inside slot, it may be `<:slotName />` .

**Comment by rwwagner90** [comment]
@lifeart would the slots be selectable? If so, what data would we show?

**Comment by lifeart** [comment]
Yes, I think slots should be selectable, and we could show slot's position arguments

`{{yield (hash a=1) "foo" "bar" to="heder"}}`

`[ {a: 1}, "foo", "bar" ]`

**Comment by chancancode** [comment]
I would like to see this, but right now we don't really have the information to do it from the debug render tree. So the first step would be to make that work, and then we can work out the display format here

**Comment by lifeart** [comment]
@chancancode follow-up request - have ability to see "if"s and "each" blocks (areas)
